src/com/bankapp/Users/Customer.java

```diff
@@ -20,16 +19,28 @@ public Customer (int ID, String name, String phoneNumber,BankAccount checkingAcc
         return accounts;
     }
 
-   public BankAccount getBankAccount(int accountID){
+    public BankAccount getBankAccount(int accountID) {
 
-       for (BankAccount account : accounts)
-       {
-           if (account.getAccountID() == accountID ) {
-               return account;
-           }
-       }
+        for (BankAccount account : accounts) {
+            if (account.getAccountID() == accountID) {
+                return account;
+            }
+        }
 
-       throw new IllegalArgumentException("Not found bank account : " + accountID);
-   }
+        throw new IllegalArgumentException("Not found bank account : " + accountID);
+    }
+
+    public BankAccount getAccountType(String type){
+//        ArrayList<BankAccount>types=new ArrayList<>();
+
+        for(BankAccount accountType: accounts) {
+            if (accountType.getType().equals(type)) {
+                accounts.add(accountType);
```

Comment:
why are you using "accounts" ?

Reply:
> why are you using "accounts" ?

I getting Bank Account types from my accounts. What should you do instead? Creating another Array List doesn't make sense I think.

Reply:
I believe inside the method you need an empty ArrayList to add a specific type of account then return